### PR TITLE
✨ confirm delete collection modal

### LINKED ISSUE #7982: Confirm deletion of collection
after:
- https://github.com/kodadot/nft-gallery/pull/7960

there is currently no step between you clicking the button of delete and signing burning your collection 
image
image

image
- I would also rename this to Delete collection
it will be more clear what its about

design added in handoff, small things https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1287%3A18959&mode=design&t=eidhg2mWSkXNBA35-1

- its the same component as in migration go to home confirmation

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review
- @exezbcz missing hover color on red button. which color would you like

## Context

- [x] Closes #7982
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

## Screenshot 📸

- [x] My fix has changed UI

image

### <samp>🤖 Generated by Copilot at fef4bb9</samp>

Added a confirmation modal for deleting a collection and its NFTs in `HeroButtons.vue`, using the `NeoModal` component and localization strings from `locales/en.json`.

### <samp>🤖 Generated by Copilot at fef4bb9</samp>

> _Sing, O Muse, of the skillful coder who devised_
> _a cunning modal, `NeoModal`, to guard the treasures_
> _of the collections and their NFTs, the shining works_
> _of digital art, from rash and heedless deletion._

### CODE REVIEW

File: components/collection/HeroButtons.vue [hunk 1]
```diff
@@ -86,6 +86,35 @@
         </div>
       </div>
     </NeoModal>
+    <NeoModal
```
Comment (roiLeo) [inline components/collection/HeroButtons.vue:89]: create a new component? (`ConfirmDeleteModal`)

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Comment by roiLeo** [comment]
Note: 
- > This will permanently remove your collection and NFTs. Your deposit will be refunded.

Technically user will have to burn each nft before he can burn collection, we don't have the full burn feature yet.

- bold text for action button is new

**Review by Jarsen136 — APPROVED** [review]

**Comment by AshutoshSingh00001** [comment]
> **Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).
> 
> 👇 __ Let's make a quick check before the contribution.
> 
> ## PR Type
> * [ ]  Bugfix
> * [x]  Feature
> * [ ]  Refactoring
> 
> ## Needs Design check
> * @exezbcz please review
> * @exezbcz missing hover color on red button. which color would you like
> 
> ## Context
> * [x]  Closes [Confirm deletion of collection  #7982](https://github.com/kodadot/nft-gallery/issues/7982)
> * [ ]  Requires deployment <snek/rubick/worker>
> 
> #### Did your issue had any of the "$" label on it?
> * [x]  My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
> 
> ## Screenshot 📸
> * [x]  My fix has changed UI
> 
> image
> 
> ## Copilot Summary
> ### 🤖 Generated by Copilot at [fef4bb9](https://github.com/kodadot/nft-gallery/commit/fef4bb90213ad5a6bd4b45dcee1c5fde240b421f)
> Added a confirmation modal for deleting a collection and its NFTs in `HeroButtons.vue`, using the `NeoModal` component and localization strings from `locales/en.json`.
> 
> ### 🤖 Generated by Copilot at [fef4bb9](https://github.com/kodadot/nft-gallery/commit/fef4bb90213ad5a6bd4b45dcee1c5fde240b421f)
> > _Sing, O Muse, of the skillful coder who devised_
> > _a cunning modal, `NeoModal`, to guard the treasures_
> > _of the collections and their NFTs, the shining works_
> > _of digital art, from rash and heedless deletion._

there is a fees of 0.25 Dot charged on polkadot asset hub  for destroy collection
unless of using delete as button text we can use destroy collection as text this keyword is used in the nft pallet

**Comment by yangwao** [comment]
Thanks!
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 5.02 USD/DOT ~ 5.976 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0xf90bb910316fc6148bfb2ad2a12ff6d8326196da10d7522ac639f6846d1546e4](https://polkadot.subscan.io/extrinsic/0xf90bb910316fc6148bfb2ad2a12ff6d8326196da10d7522ac639f6846d1546e4)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
@daiagi let's resolve conflicts and merge it 💪

**Pull request merged**